README.md

```diff
@@ -6,6 +6,16 @@ The bucc command line utility allows for easy bootstrapping of the BUCC stack (B
 
 ### Prepare the Environment
 
+#### Ubuntu 18.04/18.10
+
+1.  Make sure development environment is setup with proper utils and libraries:
+
+```
+sudo apt install libssl-dev libreadline-dev libgdbm-dev gcc make g++ ruby-dev zlibc zlib1g-dev zlib1g
+```
+
```

Comment:
Maybe better to just mention that the bosh cli also has additional dependencies https://bosh.io/docs/cli-v2-install/#additional-dependencies since that list might change it's probably better to link then to create a snapshot of that list here